README.md

```diff
@@ -1,5 +1,9 @@
 # ohsome API
 
+> :information_source: 2021 User Survey :information_source:<br>
+> The organisation behind ohsome and the OSHDB (<a href="https://heigit.org">HeiGIT</a>) will be having an organisational review. To ensure that we can continue to provide developing software and providing services in the future, we would like to invite you to take part in a <a href="https://survey.heigit.org/653841" target="_blank">user survey</a>. Thanks for your participation!
```

Comment:
```suggestion
> The organisation behind the ohsome API (<a href="https://heigit.org">HeiGIT</a>) will be having an organisational review. To ensure that we can continue to provide developing software and providing services in the future, we would like to invite you to take part in a <a href="https://survey.heigit.org/653841" target="_blank">user survey</a>. Thanks for your participation!
```

Reply:
:thinking: with `ohsome` I meant all of the ohsome platform, not only the ohsome API. But fair enough it sounds a little bit weird if I write `ohsome and the ohsome API`.